README.md

```diff
@@ -56,6 +56,46 @@ Content Checks:
  - X-XSS-Protection Header: false
 ```
 
+
+Installing directly on Ubuntu 18.04:
+===========================
+1) [Install rbenv 2.6.5](https://gorails.com/setup/ubuntu/18.04#ruby-rbenv) (if you don't already have it, make sure to use 2.6.5 not 2.7.1)
+2) Install bundler `gem install bundler`
+4) Clone the ident repository `git clone https://github.com/intrigueio/intrigue-ident.git`
+5) Install the ident gem dependencies `bundle install`
+6) Now use the check utility to test. Example below:
+```
+$ bundle exec ruby ./util/ident.rb -v -u https://xyz.com:443
```

Comment:
just one edit, can we truncate this since it's already shown above?

Reply:
Yes, truncating makes sense.